perfmetrics/scripts/build_ml.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+#To run the script
+#>> ./build.sh <start_time> <end_time>
```

Comment:
rename this file to populate_metrics and other file populate_vm_metrics

Reply:
Done.